core/ir/encode_api.h

```diff
@@ -150,21 +150,23 @@ dr_get_isa_mode(void *drcontext);
 
 DR_API
 /**
- * AArch64 Scalable Vector Extension's vector length in bits is one of:
- * 128 256 384 512 640 768 896 1024 1152 1280 1408 1536 1664 1792 1920 2048
+ * - AArch64 Scalable Vector Extension's vector length in bits is one of:
+ *   128 256 384 512 640 768 896 1024 1152 1280 1408 1536 1664 1792 1920 2048
+ * - RISC-V Vector Extension's vector length in bit is from 64 to 65536 in the
+ *   power of 2.
  * Returns whether successful.
  * TODO i#3044: This function will only allow setting vector length if not
- * running on SVE.
+ * running on SVE or RVV.
  */
 bool
-dr_set_sve_vector_length(int vl);
```

Comment:
A compatibility change should bump the version per https://dynamorio.org/page_new_release.html#autotoc_md232.  It should be bumped from 10.92 to 10.93.  See b4909144c3 the last PR to bump it (to .92) to see the places to change: I believe it's 3 files, 2 .yml files and the top-level CMakeLists.txt.